lib/internal/process/execution.js

```diff
@@ -84,38 +84,19 @@ function evalScript(name, body, breakFirstLine, print, shouldLoadESM = false) {
       evalModuleEntryPoint(body, print);
   }
 
-  const runScript = () => {
-    // Create wrapper for cache entry
-    const script = `
-      globalThis.module = module;
-      globalThis.exports = exports;
-      globalThis.__dirname = __dirname;
-      globalThis.require = require;
-      return (main) => main();
-    `;
-    globalThis.__filename = name;
-    RegExpPrototypeExec(/^/, ''); // Necessary to reset RegExp statics before user code runs.
-    const result = module._compile(script, `${name}-wrapper`)(() => {
-      const compiledScript = compileScript(name, body, baseUrl);
-      return runScriptInThisContext(compiledScript, true, !!breakFirstLine);
-    });
-    if (print) {
-      const { log } = require('internal/console/global');
-
-      process.on('exit', () => {
-        log(result);
-      });
-    }
-
-    if (origModule !== undefined)
-      globalThis.module = origModule;
-  };
+  const evalFunction = () => runScriptInContext(name,
+                                                body,
+                                                breakFirstLine,
+                                                print,
+                                                module,
+                                                baseUrl,
+                                                undefined,
+                                                origModule);
```

Comment:
What the heck is happening here?

```suggestion
  const evalFunction = () => runScriptInContext(
    name,
    body,
    breakFirstLine,
    print,
    module,
    baseUrl,
    undefined,
    origModule,
  );
```

Reply:
Its the formatter 🤷 I run `make lint-js-fix` and it formats it that way